clients/src/main/java/org/apache/kafka/common/utils/AppInfoParser.java

```diff
@@ -106,8 +106,8 @@ private static void unregisterMetrics(Metrics metrics) {
     public static class AppInfo implements AppInfoMBean {
 
         public AppInfo() {
-            log.info("Kafka version : " + AppInfoParser.getVersion());
-            log.info("Kafka commitId : " + AppInfoParser.getCommitId());
+            log.info("Kafka version : {}", AppInfoParser.getVersion());
+            log.info("Kafka commitId : {}", AppInfoParser.getCommitId());
```

Comment:
Existing issue: space after colon. No need for `()` in `getVersion` and `getCommitId`.